### PR TITLE
dockerize backend

### PR DESCRIPTION
**Pull request by brianfelix000**
completed dockerization

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM node:12.13.1-alpine
```
Comment (chejazi) [inline Dockerfile:1]: For this repo in particular we want node 14

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+WORKDIR /backend
+
+COPY package.json package-lock.json ./
+
+COPY build ./build
+
+RUN apk --no-cache add --virtual builds-deps build-base python
+RUN npm install
+
+EXPOSE 3000
```
Comment (chejazi) [inline Dockerfile:13]: I think right now the app serves on port 8000 – do these need to be the same?

File: Dockerfile (hunk 1 above)
Comment (brianfelix000) [reply Dockerfile:1]: @chejazi I will update that

File: Dockerfile (hunk 2 above)
Comment (brianfelix000) [reply Dockerfile:13]: @chejazi currently the app is running at 3000.

**Pull request merged**